src/server/src/main/java/io/cassandrareaper/resources/RepairRunResource.java

```diff
@@ -604,9 +604,9 @@ public Response abortRepairRunSegment(@PathParam("id") UUID repairRunId, @PathPa
    * @return all know repair runs for a cluster.
    */
   @GET
-  @Path("/cluster/{cluster_name}")
+  @Path("/cluster/{clusterName}")
```

Comment:
Suggestion: I think this might be the wrong way around. Generally shouldn't URI query strings use snake case? Would it be better to change this in the UI?

Issue: Also, as this is a breaking change to the API I'm not sure what our approach should be here. Does the API have a versioning policy?

Reply:
This one is a pathParam, so it doesn't matter that much. The query param from listRepairRuns on the other hand could indeed be a breaking change.
Let me see if I can turn this around.